routers/web/web.go

```diff
@@ -648,7 +656,12 @@ func registerRoutes(m *web.Route) {
 			})
 			addWebhookEditRoutes()
 		}, webhooksEnabled)
-	}, reqSignIn, ctxDataSet("PageIsUserSettings", true, "AllThemes", setting.UI.Themes, "EnablePackages", setting.Packages.Enabled))
+	}, reqSignIn, ctxDataSet(
+		"PageIsUserSettings", true,
+		"AllThemes", setting.UI.Themes,
+		"EnablePackages", setting.Packages.Enabled,
+		"UserModules", &setting.User,
```

Comment:
It seems an over-engendering. Only `account.tmpl` needs such variable, it doesn't make sense to pass it for a lot of handlers.